daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v1.md

```diff
@@ -83,6 +83,22 @@ Authenticating with Microsoft Entra ID is supported with Azure Database for Post
 | `azureClientId` | N | Client ID (application ID) | `"c7dd251f-811f-…"` |
 | `azureClientSecret` | N | Client secret (application password) | `"Ecy3X…"` |
 
+### Authenticate using AWS IAM
+
+Authenticating with AWS IAM is supported with all versions of PostgreSQL type components.
```

Comment:
pls add that they must exist too. I know it's implied, but better to be explicit in our docs :)